packages/patternfly-4/react-core/src/components/LoginPage/examples/SimpleLoginPage.js

```diff
@@ -81,6 +97,26 @@ class SimpleLoginPage extends React.Component {
         </ListItem>
       </React.Fragment>
     );
+
+    const loginForm = (
+      <LoginForm
+        usernameLabel="Username"
+        usernameValue={this.state.usernameValue}
+        onChangeUsername={this.handleUsernameChange}
+        usernameHelperTextInvalid="Unknown Username"
+        isValidUsername
```

Comment:
Can you tie this to a state variable, and maybe in the handleUsernameChange check if username is less than 3 chars mark it as invalid? And something similar for password. So we can play with invalid state in the example

Reply:
Can you open an enhancement issue to add this so we can get the form merged.  the invalid state is really a function of the form.